services/src/main/java/org/keycloak/broker/oidc/OIDCIdentityProviderConfig.java

```diff
@@ -103,7 +103,7 @@ public void setBackchannelSupported(boolean backchannel) {
 
     public boolean isDisableUserInfoService() {
         String disableUserInfo = getConfig().get("disableUserInfo");
-        return disableUserInfo == null ? false : Boolean.valueOf(disableUserInfo);
+        return Boolean.parseBoolean(disableUserInfo);
```

Comment:
This doesn't count for disableUserInfo could be null

Reply:
Maybe I'm wrong, but javadoc Boolean.parseBoolean says:

> Parses the string argument as a boolean.  The {@code boolean}
     * returned represents the value {@code true} if the string argument
     * is not {@code null} and is equal, ignoring case, to the string
     * {@code "true"}. <p>

The code Boolean.parseBoolean is:
`return ((s != null) && s.equalsIgnoreCase("true"));`

Reply:
You're correct. I read the JavaDoc now, but to be honest it's not fully clear what the behaviour is with null, so I checked and it does indeed return false.